runtime/serde/serde-form-url/common/src/aws/smithy/kotlin/runtime/serde/formurl/FormUrlSerializer.kt

```diff
@@ -130,6 +131,12 @@ private class FormUrlStructSerializer(
         serializeInstant(value, format)
     }
 
+    override fun field(descriptor: SdkFieldDescriptor, value: Document) {
+        throw SerializationException(
+            "cannot serialize field ${descriptor.serialName} to unsupported Document type in form-url"
```

Comment:
the "cannot serialize field to unsupported Document type" is a bit confusing as written.

Perhaps
> cannot serialize field ${descriptor.serialName}; Document type is not supported by form-url encoding
?